app/controllers/api/v1/users_controller.rb

```diff
@@ -16,6 +16,14 @@ def find
     render json: user_to_response(user), status: 200
   end
 
+  def request_otp
+    user = User.find(request_otp_id_param)
+    user.set_otp
+    user.save
+    SmsNotificationService.new(user).send_request_otp_sms
```

Comment:
Since this is essentially the same functionality as reset_otp, should we move this to the User model?